package.json

```diff
@@ -23,29 +38,44 @@
     "gulp-sort": "2.0.0",
     "gulp-wp-pot": "2.4.2",
     "gulp-zip": "4.2.0",
+    "husky": "4.2.5",
     "jsdoc": "3.6.4",
+    "lint-staged": "10.2.2",
     "prettier": "npm:wp-prettier@1.19.1",
     "wp-hookdoc": "0.2.0"
   },
-  "dependencies": {
-    "@woocommerce/components": "4.0.0",
-    "@wordpress/components": "9.4.1",
-    "@wordpress/i18n": "3.11.0",
-    "interpolate-components": "1.1.1",
-    "select2": "4.0.5"
-  },
   "scripts": {
     "build": "cross-env NODE_ENV=production npx calypso-build",
     "build:docs": "rm -rf hookdocs/ && jsdoc -c hookdoc-conf.json",
-    "lint-css": "wp-scripts lint-style **/*.scss",
-    "lint-css:fix": "npm run lint-css -- --fix",
-    "lint-js": "wp-scripts lint-js --ext js,jsx assets",
```

Comment:
Maybe we could leave these arguments here and call `wp-scripts lint-js` directly in the `lint-staged` config, so `npm run lint-js` is still useful running the checks manually. (And similarly for CSS)

Reply:
The thing is that the lint-staged command passes the filename as the argument, so if we leave the arguments here the command is executed like `wp-scripts lint-js --ext js,jsx assets <1st file with changes> <2nd file with changes>` which means that we end up running the linter for all files in every commit.

We could have two commands one with the `assets` argument and one without but it might be excessive. WDYT?

Reply:
What I meant was adding the commands like this (assuming it works):
```
"lint-staged": {
    "*.{js,jsx}": [
      "wp-scripts format-js",
      "wp-scripts lint-js"
    ],
```
This way the `lint-js` command in the `scripts` block would not be involved.

Reply:
Ah sorry I misunderstood that. 568d5d4830a00d2477f130f7ca4e08e1170085e2